client/my-sites/subscribers/components/subscriber-data-views/style.scss

```diff
@@ -1,26 +1,159 @@
+@import '@wordpress/base-styles/variables';
 @import '@wordpress/base-styles/breakpoints';
+@import "~@wordpress/base-styles/mixins";
 @import '@wordpress/dataviews/build-style/style.css';
 
-.subscribers.subscribers--dataviews.main {
-    .navigation-header {
-        padding: 0 48px;
+$padding-standard: 16px;
 
-        @media ( max-width: $break-small ) {
-            padding: 0 24px;
+body.is-section-subscribers.is-subscribers-dataviews {
+    overflow: hidden;
+    
+    .layout:not(.has-no-sidebar) {
+        .layout__content {
+            background: var(--studio-gray-0);
+            padding: calc(var(--masterbar-height) + #{$padding-standard})
+                 $padding-standard
+                 $padding-standard
+                 calc(var(--sidebar-width-max) + #{$padding-standard});
         }
     }
-}
 
-.subscriber-data-views {
-	.subscriber-profile.subscriber-profile--compact {
-		@media ( max-width: $break-xlarge ) {
-            max-width: 225px;
-        }
+    .subscribers.main {
+        max-width: none;
+        padding: 0;
+        margin: 0;
+
+        .subscriber-data-views {
+            display: flex;
+            flex-direction: row;
+            flex: 1 1 auto;
+            position: relative;
+            gap: 24px;
+            height: calc(
+                100vh - var(--masterbar-height) - #{$padding-standard * 2}
+            );
+            overflow: hidden;
+        
+            @media ( max-width: $break-small ) {
+                padding: $padding-standard 0;
+            }
+        
+            button {
+                cursor: pointer;
+                max-width: 100%;
+            }
+        
+            .subscriber-data-views__square-avatar {
+                border-radius: 0;
+                cursor: pointer;
+            }
+        
+            &__list {
+                flex: 1 1 auto;
+                transition: width 0.15s ease-in-out;
+                width: 100%;
+                background: var( --color-surface );
+                /* stylelint-disable-next-line scales/radii */
+                border-radius: 8px;
+                overflow: hidden;
+                display: flex;
+                flex-direction: column;
+        
+                .navigation-header {
+                    padding: $padding-standard 48px;
+        
+                    // 430px is the DataViews padding breakpoint.
+                    @media ( max-width: 430px ) {
+                        padding: $padding-standard 24px;
+                    }
+
+                    @media ( min-width: $break-small ) {
+                        .gridicon {
+                            display: none;
+                        }
+                    }
+                }
+
+                &-item {
```

Comment:
I'm re-creating the DataViews list styles here. I tried using `combinedFields` instead to put the avatar and name in one column, but then it wasn't sortable.